### ISSUE TITLE
Deleted isettools/metrics

### ISSUE THREAD

**Issue by DavidBrainard** [isetbio/isetbio#312]
I deleted isettools/metrics.  This contained code for an ISO12233 metric, which seemed very camera oriented.  One of the two routines used the sensor structure.

Posting this so if it was a mistake, we can put it back now rather than be mystified in the future.
But will immediately close the issue, because I suspect deleting was the right move.

**Comment by wandell** [comment]
Oops.  I just put it there because Trisha uses it as part of her sceneEye work.  The tool has value in specifying a metric for the Oculus team.

**Comment by DavidBrainard** [comment]
OK, that's fine. But the ieISO... version has calls to the sensor object, so perhaps she is just using the lower level one?  That one might be fine in the sense that it can run in isetbio.

**Comment by tlian7** [comment]
Hi guys,

I was using the function "ISO12233" specifically to measure the MTF of the retinal image when ray-tracing. This function is located in iset/metrics/ISOISO12233.m, and takes in an RGB image, so it seems compatible with ISETBIO.

**Comment by DavidBrainard** [comment]
Yes, that is the "low-level" one.  OK, let's keep that and get rid of ieISO12233 unless there is some reason that is needed in the long run (it will require some work).

**Comment by DavidBrainard** [comment]
I put ISO12233.m back, and left out ieISO12233.  The latter is entwined with the sensor object and doesn't seem like it will do much good in isetbio.

Trisha, if this works for you, then please close issue.

You might also see if you can clean up the examples in the source a little.  It would be nice to have one that runs without requiring user input.  And, the first one throws a warning.  And, it isn't clear what to select in interactive mode so that it is happy.  So a little tuning and commenting of the examples would be great.  I don't know enough about what the routine does to handle this.
